workflow/rocoto/gefs_tasks.py

```diff
@@ -218,14 +220,21 @@ def _atmosoceaniceprod(self, component: str):
         history_path = self._template_to_rocoto_cycstring(self._base[history_path_tmpl], {'MEMDIR': 'mem#member#'})
         deps = []
         data = f'{history_path}/{history_file_tmpl}'
-        dep_dict = {'type': 'data', 'data': data, 'age': 120}
-        deps.append(rocoto.add_dependency(dep_dict))
```

Comment:
I would like to clarify the issue here with an example which may help explain why I removed this dependency:

Let's say that `FHOUT_ICE_GFS=24 `and `cyc=12`. For the ice_prod task, we have an fhr variable: `<var name="fhr">024 048 072 096 120</var>`
If we were to keep this filename dependency
(`<datadep age="120"><cyclestr>&ROTDIR;/gefs.@Y@m@d/@H/mem#member#/model_data/ocean/history/gefs.ocean.t@Hz.24hr_avg.f#fhr#.nc</cyclestr></datadep>`), the dependency would never be satisfied because these lead hours in the fhr variable are not written in the ice hist filenames. The ice hist filenames would have lead hours 36, 60, etc.

Reply:
Since the fhr variable needs to be the same for t00z, t06z, t12z, and t18z, I couldn't immediately think of a better way to do this.

Reply:
`_get_forecast_hours()` can be updated in the same manner we are updating the `CICE_OUTPUT_FH` generation in `predet`. This would also provide the correct forecast hours for the products job, which I think are needed as well anyway.

Reply:
Ok, I'll try to accomplish this by updating `_get_forecast_hours()` and remove the need for `check_ice_netcdf.sh`.